src/NuGet/Microsoft.CodeAnalysis.Build.Tasks.nuspec

```diff
@@ -6,6 +6,8 @@
     <description>
         Contains the build task and targets used by MSBuild to run the C# and VB compilers.
         Supports using VBCSCompiler on Windows.
+
+        Source Code $commitUrl$
```

Comment:
Perhaps say "this package was built from the source at <url>" so it's at least not a random footnote?